src/components/ReportActionItem/IOUAction.js

```diff
@@ -71,6 +72,10 @@ const defaultProps = {
 const IOUAction = (props) => {
     const hasMultipleParticipants = props.chatReport.participants.length > 1;
     const onIOUPreviewPressed = () => {
+        if (!doesThreadExistForReportActionID(props.action.reportActionID)) {
+            // optimistically create reportID
+            // pass it along to OpenReport since that is build to create a thread when needed, not sure what to do about the navigation stuff below...
```

Comment:
The navigation logic is likely going to change, I wouldn't worry too much just yet.

Reply:
okay coolio thanks Jules!